docker/MOL/docker-compose.yml

```diff
@@ -0,0 +1,137 @@
+version: "2.4"
+
+volumes:
+  database:
+    name: mol-db
+  blockchain:
+    name: mol-data
+  tipbot-blockchain:
+    name: mol-tipbot-data
+  tipbot-database:
+    name: mol-tipbot-db
+
+services:
+
+  indexer:
+    container_name: mol-indexer
+    image: blockcore/indexer:0.2.39
+    mem_limit: 2048m
+    cpus: 1.500
+    environment:
+      VIRTUAL_HOST: mol.indexer.blockcore.net
+      VIRTUAL_PORT: 9910
+      VIRTUAL_PROTO: http
+      VIRTUAL_NETWORK: proxy
+      LETSENCRYPT_HOST: mol.indexer.blockcore.net
+      LETSENCRYPT_EMAIL: admin@blockcore.net
+      ASPNETCORE_URLS: https://+:9910
+      Logging__LogLevel__Default: Debug
+    command: ["--chain=MOL"]
+    restart: unless-stopped
+    depends_on:
+      mongo:
+        condition: service_healthy
+      chain:
+        condition: service_healthy
+    networks:
+      - molxnetwork
+      - proxy
+
+  chain:
+    container_name: mol-chain
+    image: blockcore/node-multi:1.1.50
+    mem_limit: 6144m
+    cpus: 2.000
+    healthcheck:
+      test: curl --fail http://localhost:22103/api/Node/status || exit 1
+      interval: 10s
+      retries: 5
+      start_period: 5s
+      timeout: 10s
+    command: ["--chain=mol", "-server=1", "-txindex=1", "-iprangefiltering=0", "-rpcallowip=10.202.0.0/8", "-rpcallowip=10.201.0.0/8", "-rpcallowip=192.168.0.0/8", "-rpcallowip=172.0.0.0/8", "-rpcbind=0.0.0.0", "-apiuri=http://0.0.0.0:22103", "-rpcuser=rpcuser", "-rpcpassword=rpcpassword"]
+    restart: unless-stopped
+    stop_grace_period: 15m
+    volumes:
+      - blockchain:/root/.blockcore
+    ports:
+      - 22105:22105 # Make this a public node.
+    networks:
+      - molnetwork
+
+  mongo:
+    container_name: implx-mongo
+    image: mongo:5.0.12
+    mem_limit: 3072m
```

Comment:
Probably enough with 2GB.

Reply:
I made a copy of the current IMPLX settings. I will reduce